### PR TITLE
post migration changes

### PR DESCRIPTION
**Pull request by ramyakrishnai**
## Proposed changes
Jira: https://jira.newfold.com/browse/PRESS0-1157

Post Migration steps *****
Given when the migration is successful
When customer is SSO'ed to wp-admin dashboard
Then we will need to display the following options with -> as part of "Next Steps"
"Update your Nameserver to BH"
"Connect the site to your Domain"
And on click of the -> Help Center slider shall open with information for the following questions
How do I update my nameserver to BH?
How do I connect my site to the Domain ?

Need help with these steps ? "View guide" link to https://www.bluehost.com/help

Screenshot 2024-04-25 at 3 05 21 PM

## Type of Change

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

- [ ] I have read the [CONTRIBUTING](https://github.com/bluehost/.github/blob/master/.github/contributing.md) doc
- [ ] Linting and tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: src/configs/OnboardingList.config.js
Comment (aratidgr8) [inline src/configs/OnboardingList.config.js:69]: Avoid using `NewfoldRuntime?.sdk`

File: src/configs/OnboardingList.config.js
Comment (aratidgr8) [inline src/configs/OnboardingList.config.js:125]: Translations not needed?

File: src/configs/OnboardingList.config.js
Comment (aratidgr8) [inline src/configs/OnboardingList.config.js:109]: Translations not needed?

File: src/configs/OnboardingList.config.js
Comment (aratidgr8) [inline src/configs/OnboardingList.config.js:71]: Can we avoid redundant code of constant `check_url_match` in `src/configs/OnboardingList.config.js` and `src/components/OnboardingList.js`

Define once & import as per need.

**Comment by aratidgr8** [comment]
@ramyakrishnai Please add more details in PR description.

File: src/components/OnboardingList.js
Comment (ramyakrishnai) [reply src/components/OnboardingList.js:66]: currently in newfoldRuntime, we're not getting ecommerce out of sdk, only once the ecommerce is out side of sdk, we can add this change

File: src/configs/OnboardingList.config.js
Comment (ramyakrishnai) [reply src/configs/OnboardingList.config.js:69]: currently in newfoldRuntime, we're not getting ecommerce out of sdk, only once the ecommerce is out side of sdk, we can add this change

File: src/components/OnboardingList.js
Comment (aratidgr8) [reply src/components/OnboardingList.js]: Why can't we use `NewfoldRuntime?.plugin?.brand` we  can also skip `toLowerCase()` if we use that variable.

File: src/configs/OnboardingList.config.js
Comment (aratidgr8) [reply src/configs/OnboardingList.config.js:69]: Why can't we use `NewfoldRuntime?.plugin?.brand` we  can also skip `toLowerCase()` if we use that variable.

File: src/configs/OnboardingList.config.js
Comment (aratidgr8) [reply src/configs/OnboardingList.config.js:142]: @ramyakrishnai plz cross-check this one as well

File: src/components/OnboardingList.js
Comment (aratidgr8) [inline src/components/OnboardingList.js:25]: Can you pass `className` as props

File: src/components/OnboardingList.js
Comment (aratidgr8) [inline src/components/OnboardingList.js:65]: we use `NewfoldRuntime?.plugin?.brand` we can also skip toLowerCase() if we use that variable.

File: src/components/OnboardingScreen.js
Comment (aratidgr8) [inline src/components/OnboardingScreen.js:99]: Translations need to be added

File: src/components/OnboardingScreen.js
Comment (aratidgr8) [inline src/components/OnboardingScreen.js:107]: Translations need to be added

File: src/configs/OnboardingList.config.js
Comment (aratidgr8) [inline src/configs/OnboardingList.config.js:68]: check if `brandName` variable could also be added to constants, and imported at necessary places

File: src/components/OnboardingList.js
Comment (ramyakrishnai) [reply src/components/OnboardingList.js]: We can't use plugin, becuase when bluehost changed it's brand like bluehost-us, hostgator_latam, at that time, this value wil not work

File: src/configs/OnboardingList.config.js
Comment (ramyakrishnai) [reply src/configs/OnboardingList.config.js:69]: We can't use plugin, becuase when bluehost changed it's brand like bluehost-us, hostgator_latam, at that time, this value wil not work

File: src/components/OnboardingScreen.js
Comment (ramyakrishnai) [reply src/components/OnboardingScreen.js:107]: translations are added in the variable which is being used here

File: src/components/OnboardingScreen.js
Comment (ramyakrishnai) [reply src/components/OnboardingScreen.js:110]: translations are added in the variable which is being used here

File: src/components/OnboardingScreen.js
Comment (ramyakrishnai) [reply src/components/OnboardingScreen.js:99]: translations are added in the variable which is being used here

File: src/components/OnboardingList.js
Comment (ramyakrishnai) [reply src/components/OnboardingList.js:25]: I'm passing classname as prop which is being destructed as state from props, these values are coming from onboarding.config.js

File: src/configs/OnboardingList.config.js
Comment (aratidgr8) [reply src/configs/OnboardingList.config.js]: As discussed on call we would go `split` way and add a `TODO`

File: src/components/OnboardingList.js
Comment (aratidgr8) [reply src/components/OnboardingList.js]: As discussed on call we would go split way and add a TODO

File: src/configs/Utility.js
Comment (aratidgr8) [inline src/configs/Utility.js:5]: We will also have to check for `underscore` for `hostgator_latam` isn't it?

**Review by aratidgr8 — CHANGES_REQUESTED** [review]

File: src/configs/OnboardingList.config.js
Comment (aratidgr8) [inline src/configs/OnboardingList.config.js:138]: This can be moved out of `sdk` ?

File: includes/Data/Brands.php
Comment (aratidgr8) [inline includes/Data/Brands.php:37]: We were to remove all `Brands.php` changes right?

Comment (ramyakrishnai) [reply includes/Data/Brands.php:37]: yeah, the changes were removed already

**Review by aratidgr8 — APPROVED** [review]

**Comment by ramyakrishnai** [comment]
@aratidgr8 can I know why is this PR labeled as `Do not merge` even after getting your approval for this PR ?

**Review by manikantakailasa — APPROVED** [review]

**Review by manikantakailasa — APPROVED** [review]

**Pull request merged**
